application-services/custom/camera-management/appcamera/routes.go

```diff
@@ -257,6 +254,19 @@ func (app *CameraManagementApp) ptzRoute(w http.ResponseWriter, req *http.Reques
 	var res dtosCommon.BaseResponse
 	var err error
 
+	// Get pan/tilt x,y range for a particular device
+	ptzConfigs, err := app.getPTZConfiguration(deviceName)
```

Comment:
It would be nice to have some caching mechanism where this only queries the camera if it has not queried it before. lazy-load style.  Could be as simple as a map<deviceName, Response> and a mutex.